### PR TITLE
Improve local script

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
Improves setting up the sample application.

### CODE REVIEW

File: script/local [hunk 1]
```diff
@@ -51,4 +50,4 @@ RAILS_COMMANDS = %w{generate console server dbconsole g c s runner}
 args = RAILS_COMMANDS.include?(ARGV[0]) ? ["rails", ARGV].flatten : ARGV
 
 # Run the command
-exec "cd test-rails-app && GEMFILE=../Gemfile #{args.join(" ")}"
+exec "cd test-rails-app && #{args.join(" ")}"
```
Comment (timoschilling) [inline script/local]: Can you please explain why this change (Line 34, 53)?

File: tasks/test.rake [hunk 2]
```diff
@@ -7,7 +7,6 @@ task :setup, :parallel do |t, opts|
     system 'mkdir -p spec/rails'
     args = %w[
       -m\ spec/support/rails_template.rb
```
Comment (timoschilling) [inline tasks/test.rake]: This line should be `-m spec/support/rails_template_with_data.rb` like in `script/local`

**Comment by timoschilling** [comment]
I like to see min 2 reviewers who has tested it local before merging this.

File: tasks/test.rake (hunk 2 above)
Comment (deivid-rodriguez) [inline tasks/test.rake]: Why? This is used for tests only, I think it's better like it is now... without data. I don't think the tests would even pass if we seed the test DB.

Comment (timoschilling) [inline tasks/test.rake]: Sorry I miss read the code, I only looked on the code formating, with removing the `\`

File: script/local (hunk 1 above)
Comment (deivid-rodriguez) [inline script/local]: Yes, I should probably squash https://github.com/activeadmin/activeadmin/pull/4763/commits/f495ef6d505fbaa876fbf1be8a293b6a39a8510f and https://github.com/activeadmin/activeadmin/pull/4763/commits/ad6707de6e93e52b1dabde2d6aedad19c8e4d7f1 since they are part of the same thing. There's plenty of reasons to do this:Previously I couldn't create the sample application because of the error in https://github.com/activeadmin/activeadmin/pull/4763/commits/ad6707de6e93e52b1dabde2d6aedad19c8e4d7f1 which is pretty self descriptive. I don't know what that `GEMFILE` environment variable does, I guess it makes the `rails` command pick up that `Gemfile`but it's definitely not the best way to go. Why create the sample application without a `Gemfile` and then point to an external `Gemfile` to make it work?

Comment (timoschilling) [inline script/local]: 👍

**Comment by deivid-rodriguez** [comment]
@timoschilling Thanks for the review. I'm fine with more people trying this out. Just note that this is unlikely to break anything since... it's already broken! :stuck_out_tongue:

File: tasks/test.rake (hunk 2 above)
Comment (deivid-rodriguez) [inline tasks/test.rake]: Ah, ok. Well, it does make sense to make escaping consistent. I'll fix it.

Comment (deivid-rodriguez) [inline tasks/test.rake]: Addressed in c22d062.
